addons/utm/wizard/utm_merge_utm_campaign.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MergeUtmCampaign(models.TransientModel):
+    """
+        Merger UTM campaigns together.
+    """
+
+    _name = 'utm.merge.utm.campaign'
+    _description = 'Merge UTM Campaigns'
+
+    @api.model
+    def default_get(self, fields):
+        """ Use active_ids from the context to fetch the UTM campaign to merge.
+        """
+        record_ids = self._context.get('active_ids')
+        result = super(MergeUtmCampaign, self).default_get(fields)
+        result['utm_campaign_ids'] = record_ids
+        result['campaign_id'] = record_ids[0]
+        return result
+
+    campaign_id = fields.Many2one('utm.campaign', string='Keep Campaign', required=True)
+    name = fields.Char(string='Campaign Name', translate=True, required=True)
+    user_id = fields.Many2one('res.users', string='Responsible', required=True)
+    stage_id = fields.Many2one('utm.stage', string='Stage', ondelete='restrict', required=True)
+    utm_campaign_ids = fields.Many2many('utm.campaign', 'merge_utm_campaign_rel', 'merge_id', 'utm_campaign_id', string='UTM Campaigns')
+
+    @api.onchange('campaign_id')
+    def _set_default(self):
+        self.name = self.campaign_id.name
+        self.user_id = self.campaign_id.user_id
+        self.stage_id = self.campaign_id.stage_id
```

Comment:
Can you try transforming that onchange into a compute by modifying fields to `store=True, readonly=False` ?
We try to remove onchange whenever possible in favor of compute methods as those are more robust.

You can check the behavior of `_compute_save_as_email` in the survey module to get a simple example.